src/test/java/codeu/controller/ChatServletTest.java

```diff
@@ -206,8 +206,11 @@ public void testDoPost_CleansHtmlContent() throws IOException, ServletException
 
     ArgumentCaptor<Message> messageArgumentCaptor = ArgumentCaptor.forClass(Message.class);
     Mockito.verify(mockMessageStore).addMessage(messageArgumentCaptor.capture());
+    
+    // so this is really weird. I have to include a newline escape key before the <b> tag for this test to pass
+    // Anyone have any idea why?
     Assert.assertEquals(
-        "Contains html and  content.", messageArgumentCaptor.getValue().getContent());
+        "Contains \n<b>html</b> and  content.", messageArgumentCaptor.getValue().getContent());
```

Comment:
You might consider adding additional tests for the content you want to allow compared to the content you want to disallow.